thinkphp/application/index/view/article/main.html

```diff
@@ -158,21 +158,24 @@ <h4 class="modal-title" id="myModalLabel">{$material->designation}</h4>
                             <p><b class="pfont">描述 :</b> {$attraction->description} </p>
                         </div>
                     </div>
-                    
+
                     {/volist}
                     <div class="clearfix"></div>
                 </div>
-                {volist name="paragraphDowns" id="_pragraph"}
-                <div class="about">
+                {volist name="paragraphDowns" id="_pragraph" key = "key"}
+                <div class="about"
```

Comment:
nth-child()
https://stackoverflow.com/questions/5428676/nth-child-doesnt-respond-to-class